File: comanda-service/src/main/java/com/fyh/comandaservice/dto/ComandaDto.java
package com.fyh.comandaservice.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.math.BigDecimal;
import java.sql.Timestamp;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ComandaDto {

    private Long id;
    private Long idClient;
    private Long idSpecialist;
    private Long idServiciu;
    private Timestamp dataCreare;
    private String status;
    private BigDecimal pret;
    private String mesajClient;

    private String numeFisierClient;
    private byte[] fisierClient;
    private Timestamp dataMaximaLivrare;
    private String mesajSpecialist;

    private String numeFisierSpecialist;
    private byte[] fisierSpecialist;

    private String titluServiciu;
    private String numeSpecialist;
    private String numeClient;
}

File: frontend/src/app/models/comanda.dto.ts
export interface ComandaDto {
  id?: number;
  idClient?: number;
  idSpecialist?: number;
  idServiciu?: number;
  dataCreare?: string;
  status?: string;
  pret?: number;
  mesajClient?: string;
  numeFisierClient?: string;
  fisierClient?: any;
  dataMaximaLivrare?: string;
  mesajSpecialist?: string;
  numeFisierSpecialist?: string;
  fisierSpecialist?: any;

  titluServiciu?: string;
  numeSpecialist?: string;
  numeClient?: string;
}
